tox.ini

```diff
@@ -3,6 +3,11 @@ envlist = py36,docs
 [pytest]
 xfail_strict = true
 testpaths = tests
+filterwarnings =
+    error::Warning
+    ignore:invalid escape sequence:DeprecationWarning
```

Comment:
What throws this one?

Reply:
I should comment those. Can I comment each line in this file? The invalid escape sequence is for using Latex inside a docstring that isn't a "raw" docstring, with the r in front. Unfortunately, networkx does this, but I think some others do too. It's still a good library. I swear. Used it for years.

Reply:
I'm pro-networkx. Don't need to convince me.